packages/owl-bot/src/core.ts

```diff
@@ -474,6 +474,32 @@ async function hasOwlBotLoop(
   return false;
 }
 
+/*
+ * Return whether or not the last commit was from OwlBot.
+ *
+ * @param owner owner of repo.
+ * @param repo short repo name.
+ * @param prNumber PR to check for commit.
+ * @param octokit authenticated instance of octokit.
+ * @returns Promise was the last commit from OwlBot?
+ */
+async function lastCommitFromOwlBot(
+  owner: string,
+  repo: string,
+  prNumber: number,
+  octokit: Octokit
+): Promise<boolean> {
+  const commits = (
+    await octokit.pulls.listCommits({
```

Comment:
Probably it's alright, but there's an edge case where there are more than 100 commits.

Reply:
I don't think this API allows for more than 100 commits, maybe just add a comment.

Reply:
I'm wrong, it can potentially support pagination, I will add note.